docs/docs/sample-scripts/spo/remove-pending-api-permission-requests.md

```diff
@@ -0,0 +1,38 @@
+# Remove pending SharePoint API permission requests
+
+Author: [Waldek Mastykarz](https://blog.mastykarz.nl/sample-script-quickly-remove-pending-sharepoint-api-permission-requests/)
+
+When building SharePoint Framework solutions connected to APIs secured with Azure Active Directory,  you'll easily end up with many pending permission requests.
+
+This script helps you to quickly remove pending SharePoint API permission requests. 
+
+```powershell tab="PowerShell Core"
```

Comment:
This is not a PowerShell script. This is a JavaScript script running on Google zx